docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -434,6 +437,38 @@ from perf counters log. The "exec type" field will contain the implementation ty
 
    MatMul_1800         EXECUTED         layerType: FullyConnected         execType: brgemm_avx512_amx_sparse_I8 realTime (ms): 0.050000  cpuTime (ms): 0.050000
 
+Multi-Threading Optimization
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+CPU inference will infer an input or multiple inputs in parallel on multiple CPU processors. 
+
+From 2023.0 release, user can use properties ``ov::inference_num_threads``, ``ov::hint::scheduling_core_type`` and ``ov::hint::enable_hyper_threading`` to limit candidate CPU resource for model inference. Then OpenVINO Runtime will perform multi-thread scheduling according to the limited candidate CPU resources.
```

Comment:
I don't see any behavior explanation for these properties - that have to be added. I would also propose to specify behavior in case value is incompatible with the platform (examples: num threads more that available cores, ECORE type on non hybrid system, hyper-threading on Ecores).

Reply:
updated